tffconnect_frontend/src/app/contexts/JWTAuthContext.js

```diff
@@ -106,14 +106,25 @@ export const AuthProvider = ({ children }) => {
         })
     }
     
-    const register = async (email, username, password) => {
-        const response = await axios.post('/api/auth/register', {
-            email,
+    const register = async (username, first_name, last_name, password) => {
+        const response = await axios.post(API_URL + '/users/signup/', {
             username,
+            first_name,
```

Comment:
Do we need these first_name and last_name.

what is the format of the json message?

Reply:
since the user model in backend has the fields first_name and last_name, I adjusted the frontend and POST api into a form that takes these keys too.